app/config.py

```diff
@@ -108,6 +108,8 @@ class Config(object):
 
     # Zendesk
     ZENDESK_API_KEY = os.getenv('ZENDESK_API_KEY')
+    ZEN_DESK_SELL_API_URL = os.getenv('ZEN_DESK_SELL_API_URL')
```

Comment:
Maybe `rstrip('/')` this?

Reply:
good suggest, I think it would be good for me to explore `urllib.parse.urljoin`, then we don't have to deal with this type of string manipulation